magic_sturdy/magic_sturdy.md

```diff
@@ -96,5 +96,66 @@ vowel side index. Most critically, this finger begins with fairly low usage and
 vowel index in a typical layout (The weighted finger movement as measured by Oxeylyzer is only 0.661). 
 
 ## Metrics
+The following stats are from [Mana](https://github.com/Zakkkk/mana). These stats are for the Magic Sturdy variant without any briefs, as in only the bigram rules. The stats are for monkeyracer (Monkeytype + Typeracer quotes) and e200 (the default Monkeytype word list of the most popular 200 words).
 
-TODO
+monkeyracer:
+```
+MagicSturdy | monkeyracer
+  v m l c p "  | b * u o , \
+  s t r d y q  $ f n e a i -
+  x q j g w ~  ~ z h ' ? .
+
+Magic rules: ao cy dy eu gy lk oa py rl sk ue yp
+
```

Comment:
Looks like the `ks` rule is missing. I would also add `tm` in the absence of `tment`, but that probably doesn't affect any stats meaningfully.